yaml/csi-driver/edge/hpe-csi-k8s-1.18.yaml

```diff
@@ -0,0 +1,598 @@
+# Configuration to deploy the HPE CSI driver compatible with
+# Kubernetes = v1.18
+#
+# example usage: kubectl create -f <this_file>
+
+---
+
+#############################################
+############  HPE Node Info CRD  ############
+#############################################
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: hpenodeinfos.storage.hpe.com
+spec:
+  group: storage.hpe.com
+  names:
+    kind: HPENodeInfo
+    plural: hpenodeinfos
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        hpeNodes:
+          description: List of HPE nodes configured for storage access.
+          items:
+            properties:
+              uuid:
+                description: The UUID of the node.
+                type: string
+              iqns:
+                description: List of IQNs configured on the node.
+                items:
+                  type: string
+                type: array
+              chapUser:
+                  description: The CHAP User Name
+                  type: string
+              chapPassword:
+                 description: The CHAP Password
+                 type: string
+              networks:
+                description: List of networks configured on the node.
+                items:
+                  type: string
+                type: array
+              wwpns:
+                description: List of WWPNs configured on the node.
+                items:
+                  type: string
+                type: array
+          type: array
+  version: v1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+
+---
+
+################# CSI Driver CRD ###########
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: csi.hpe.com
+spec:
+  podInfoOnMount: true
+  volumeLifecycleModes:
+  - Persistent
+  - Ephemeral
```

Comment:
I think this GA'd in 1.18 so we might not need to create the CRD ourselves any longer.

Reply:
https://kubernetes-csi.github.io/docs/csi-driver-object.html yes it did. will remove

Reply:
Please add this back Raunak..we still need to create the instance of CSIDriver for hpe. CRD itself might be there by default with GA.